pkg/test/istio.io/tasks/security/authz-http/verify-lib.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
This is OK as a temporary workaround. I'd like to have a more generic solution to match on output in the Goland test code. Later.

Reply:
Sure. I feel a bash script might still be useful if we want to refer to it directly in the task.

Reply:
The concern I have is that with this approach we'll have non-example code (e.g. assertion checking) in the script. We agreed that we should be able to embed each script into the doc later, to make sure that the tests and the doc are in sync. If we have assertion checking code in the scripts, we can't do that.

Reply:
I see, then I think the bigger issue the current authorization task doesn't have any embedded script for checking the result, instead it asks users to check the web page manually in the browser. We might want to update the doc to actually use some script to check the status and then reuse it here.

Reply:
Ah yes, not all the docs may be testable automatically. 🤔
It's probably OK to just add a comment into your script to document that this is equivalent to the user checking in the browser.